src/transformers/models/regnet/configuration_regnet.py

```diff
@@ -0,0 +1,92 @@
+# coding=utf-8
+# Copyright 2022 Meta Research, Inc. and The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" RegNet model configuration"""
+
+from ...configuration_utils import PretrainedConfig
+from ...utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+REGNET_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "regnety-40": "https://huggingface.co/zuppif/regnety-040/blob/main/config.json",
+}
+
+
+class RegNetConfig(PretrainedConfig):
+    r"""
+    This is the configuration class to store the configuration of a [`RegNetModel`]. It is used to instantiate an
+    RegNet model according to the specified arguments, defining the model architecture. Instantiating a configuration
+    with the defaults will yield a similar configuration to that of the
+    [regnety-40](https://huggingface.co/microsoft/regnety-40) architecture.
```

Comment:
To be updated.

Reply:
We usually also do [facebook/regnet-y-40](https://huggingface.co/facebook/regnet-y-40)

i.e. we include the organization as well in the name.

Reply:
thanks!